lib/pbench/test/unit/server/query_apis/test_datasets_delete.py

```diff
@@ -39,28 +39,29 @@ def fake_elastic(self, monkeypatch, map: JSON, partial_fail: bool):
         expected_results = []
         expected_ids = []
 
-        for index in map:
+        for _r, indices in map.items():
```

Comment:
Rather than iterating over `items()` and marking `_r` as unused, consider using `values()`:
```python
        for indices in map.values():
```

Ditto for `lib/pbench/test/unit/server/query_apis/test_datasets_update.py` line 40.

Reply:
Code evolution. I did use `values` in several other places, but this evolved along a divergent path. 😏